Unable to package `ABX` project

<!-- Please search existing issues to avoid creating duplicates. -->

### Description

When running `mvn clean package` on an ABX project an error is thrown.

[Description of the issue. Affected functionality / feature.]


### Steps to Reproduce

1. Create a sample project using the following script
```
mvn archetype:generate \
    -DinteractiveMode=false \
    -DarchetypeGroupId=com.vmware.pscoe.polyglot.archetypes \
    -DarchetypeArtifactId=package-polyglot-archetype \
    -DarchetypeVersion=2.44.0 \
    -DgroupId=local.corp.devops.common \
    -DartifactId=testAbxProject \
    -Dtype=abx
```
2. Run `mvn clean package`

**Expected behavior:** [What you expect to happen]
The project packages successfully.

**Actual behavior:** [What actually happens]
The following error is thrown:
`[ERROR] Failed to execute goal com.vmware.pscoe.maven.plugins:abx-package-maven-plugin:2.44.0:package (default-package) on project anothertest: Error creating ABX bundle: duplicate entry: node_modules/run-script-os/index.js -> [Help 1]`

**Reproduces how often:** 100%


**Affects Build/s:**
2.44.0

### Environment

#### Client

-   Build Tools for VMware Aria Version: 2.44.0
-   Visual Studio Code Version: 1.93.1 (Universal)
-   OS Version: MacOS Sonoma 14.5

### Failure Logs

![image](https://github.com/user-attachments/assets/b683655e-045d-4e30-8640-999de8c7b46f)

### Related issues and PRs

<!-- Link any related issues and pull requests here using #number or user/repo#number -->

### Additional Context

Please, after fixing the build issue try also pushing to make sure that the push manages to successfully deliver the created package to vRO.
`mvn clean package vrealize:push -P<profileName>`
<!-- Add any other information, configuration or screenshots that might be necessary to reproduce the issue. -->

--- a/docs/versions/latest/Release.md
+++ b/docs/versions/latest/Release.md
@@ -87,6 +87,15 @@ We were getting an error when trying to package ABX project:
 
 The ABX package created successfully.
 
+### *Add interactive installer option for importing ABX package*
+
+#### Previous Behavior
+
+ABX packages are ignored in the bundle. The Installer completes the task without installing the packages.
+
+#### New Behavior
+
+The Installer asks the user whether to install ABX packages and in case of positive response installs the packages.
 
 ## Upgrade procedure
 

--- a/docs/versions/latest/Release.md
+++ b/docs/versions/latest/Release.md
@@ -74,6 +74,20 @@ This was because of the `//` after the version id.
 
 The extra `/` is removed when forming the url now
 
+### *Unable to package ABX project*
+
+#### Previous Behavior
+
+We were getting an error when trying to package ABX project:
+```log
+[ERROR] Failed to execute goal com.vmware.pscoe.maven.plugins:abx-package-maven-plugin:2.44.0:package (default-package) on project anothertest: Error creating ABX bundle: duplicate entry: node_modules/run-script-os/index.js -> [Help 1]
+```
+
+#### New Behavior
+
+The ABX package created successfully.
+
+
 ## Upgrade procedure
 
 [//]: # (Explain in details if something needs to be done)

--- a/maven/archetypes/polyglot/src/main/resources/archetype-resources/package.json
+++ b/maven/archetypes/polyglot/src/main/resources/archetype-resources/package.json
@@ -39,7 +39,8 @@
 		"inputConstants": [],
 		"inputs": {
 			"tagKey": ""
-		}
+		},
+		"additionalProperties": []
 	},
 	"files": [
 		"out",

--- a/maven/plugins/abx/src/main/java/com/vmware/pscoe/maven/plugins/PackageMojo.java
+++ b/maven/plugins/abx/src/main/java/com/vmware/pscoe/maven/plugins/PackageMojo.java
@@ -65,9 +65,10 @@ public class PackageMojo extends AbstractMojo {
 		List<File> dirList = new ArrayList<>();
 		List<String> packageNameList = new ArrayList<>();
 		List<Artifact> depArtifacts = new ArrayList<>();
+		File distDir = new File(project.getBasedir(), "dist");
 
 		// Prepare list of artifact root folders
-		getBundlesList(project.getBasedir(), packageName, dirList, packageNameList);
+		getBundlesList(distDir, packageName, dirList, packageNameList);
 
 		int index = 0;
 		for (File dir : dirList) {
@@ -82,7 +83,7 @@ public class PackageMojo extends AbstractMojo {
 			getLog().info("ABX action name '" + newPackageName + "'");
 
 			MavenProjectPackageInfoProvider pkgInfoProvider = new MavenProjectPackageInfoProvider(project);
-			File pkgFile = new File(dir, pkgInfoProvider.getPackageName() + "." + PackageType.ABX.getPackageExtention());
+			File pkgFile = new File(directory, pkgInfoProvider.getPackageName() + "." + PackageType.ABX.getPackageExtention());
 			Package pkg = PackageFactory.getInstance(PackageType.ABX, pkgFile);
 			try {
 				this.preparePackageFile(pkg, dir, new File(pkgFile.getAbsolutePath()));

--- a/package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java
+++ b/package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java
@@ -1069,6 +1069,13 @@ public final class Installer {
 			userInput(input, Option.CS_IMPORT, "Import Code Stream packages?", true);
 		}
 		// +-------------------------------------
+		// | ABX
+		// +-------------------------------------
+		boolean hasAbxPackages = !getFilesystemPackages(PackageType.ABX).isEmpty();
+		if (hasAbxPackages) {
+			userInput(input, Option.ABX_IMPORT, "Import ABX packages?", true);
+		}
+		// +-------------------------------------
 		// | vRealize Automation (New Generation)
 		// +-------------------------------------
 		boolean hasVraNgPackages = !getFilesystemPackages(PackageType.VRANG).isEmpty();
@@ -1078,7 +1085,7 @@ public final class Installer {
 				userInput(input, Option.VRANG_DELETE_CONTENT, "Clean up Aria Automation content?", true);
 			}
 		}
-		if (input.anyTrue(Option.VRANG_IMPORT, Option.CS_IMPORT, Option.VRANG_DELETE_CONTENT)) {
+		if (input.anyTrue(Option.VRANG_IMPORT, Option.CS_IMPORT, Option.VRANG_DELETE_CONTENT, Option.ABX_IMPORT)) {
 			readVrangProperties(input);
 		}
 		if (input.anyTrue(Option.VRANG_IMPORT)) {

